### PR TITLE
add headers

### PR DESCRIPTION
**Pull request by JoseMotto-sg**
## Change description

Repository header update. 

## Type of change
- [ ] Bug fix (fixes an issue)
- [x] New feature (adds functionality)

## Related issues

> Fix [#1](No Applicable) 

### Development

- [x] Lint rules pass locally
- [x] Application changes have been tested thoroughly
- [ ] Automated tests covering modified code pass

### Security

- [x] Security impact of change has been considered
- [x] Code follows company security practices and guidelines

### Code review 

- [x] Pull request has a descriptive title and context useful to a reviewer. Screenshots or screencasts are attached as necessary
- [x] "Ready for review" label attached and reviewers assigned
- [ ] Changes have been reviewed by at least one other contributor
- [ ] Pull request linked to task tracker where applicable

### CODE REVIEW

**Review by fernandoPedraza-sg — CHANGES_REQUESTED** [review]

File: .config/.terraform-docs.yml [hunk 1]
```diff
@@ -1,10 +1,13 @@
...
+
 version: ""
 
 formatter: markdown table
 
 name: terraform-github-repository-creation
 
 content: |-
+  {{ .Header }}
   # terraform-github-repository-creation
```
[the remark this answers is not in the document]
Comment (JoseMotto-sg) [reply .config/.terraform-docs.yml:12]: done

File: .config/header.md [hunk 2]
```diff
@@ -0,0 +1,10 @@
+<p align="center">                                                                                                                                            ─╯
```
Comment (fernandoPedraza-sg) [inline .config/header.md:1]:
```suggestion
<p align="center">                                                                                                                                        ```
@JoseMotto-sg, please remove spaces and text character at the end of this line.

**Review by fernandoPedraza-sg — CHANGES_REQUESTED** [review]

File: .config/header.md [hunk 3]
```diff
@@ -0,0 +1,10 @@
+<p align="center">                                                                                                                                        ```
+  <img src="https://github.com/StratusGrid/terraform-readme-template/blob/main/header/stratusgrid-logo-smaller.jpg?raw=true" />
+  
+  <p align="center">                                                           
+    <a href="https://stratusgrid.com/book-a-consultation">Contact Us Test</a> |
```
Comment (jheison-rodriguez) [inline .config/header.md:5]: Remove ` Test`

File: README.md [hunk 4]
```diff
@@ -1,4 +1,14 @@
 <!-- BEGIN_TF_DOCS -->
+<p align="center">                                                                                                                                            ─╯                                                                                   
+  <img src="https://github.com/StratusGrid/terraform-readme-template/blob/main/header/stratusgrid-logo-smaller.jpg?raw=true" />
+  <p align="center">                                                           
+    <a href="https://stratusgrid.com/book-a-consultation">Contact Us Test</a> |
```
Comment (jheison-rodriguez) [inline README.md:5]: Remove ` Test`

**Review by jheison-rodriguez — CHANGES_REQUESTED** [review]

File: .config/header.md (hunk 3 above)
Comment (JoseMotto-sg) [reply .config/header.md:5]: Please open a suggestion directly

File: README.md (hunk 4 above)
Comment (JoseMotto-sg) [reply README.md:5]: Please open a suggestion directly

File: .config/header.md (hunk 3 above)
Comment (jheison-rodriguez) [inline .config/header.md:5]:
```suggestion
    <a href="https://stratusgrid.com/book-a-consultation">Contact Us</a> |
```

File: README.md (hunk 4 above)
Comment (jheison-rodriguez) [inline README.md:5]:
```suggestion
    <a href="https://stratusgrid.com/book-a-consultation">Contact Us</a> |
```

**Review by jheison-rodriguez — CHANGES_REQUESTED** [review]

**Pull request closed without merging**